examples/source/1.components/amp-autocomplete.html

```diff
@@ -8,7 +8,7 @@
 -->
 <!-- -->
 <!doctype html>
-<html ⚡ lang="en">
+<html ⚡ lang="en" data-css-strict>
```

Comment:
This is not an email only sample, we should put this behind a template variable that will only inject the flag if the email mode is enabled.

Reply:
this was just a mistake - email has [it's own autocomplete example](https://github.com/ampproject/amp.dev/blob/use-data-css-strict/examples/source/1.components/amp-autocomplete-email.html)

Reply:
Afaik, some examples are rewritten for email.

Reply:
I believe `amp-bind-macro` is the only example of that, I will get that one sorted shortly

Reply:
Found it, we need to add the flag here as well: https://github.com/ampproject/amp.dev/blob/future/platform/lib/format-transform/formats/amp4email.js#L26

Reply:
beauty, thanks. I would've taken me a long while to find that.

pushed and update for that and for the test suite